components/style/context.rs

```diff
@@ -383,8 +384,8 @@ impl fmt::Display for TraversalStatistics {
 impl TraversalStatistics {
     /// Computes the traversal time given the start time in seconds.
     pub fn finish<E, D>(&mut self, traversal: &D, parallel: bool, start: f64)
-        where E: TElement,
-              D: DomTraversal<E>,
+    where E: TElement,
```

Comment:
AFAIK all trait bounds must be on their on line indented by 4 after the `where`, but I don't care very much in such a massive PR if that is the only nit I come with.

Reply:
Whoops, missed it, will fix.